### ISSUE TITLE
Issue in creating changelog

### ISSUE THREAD

**Issue by jonniebigodes** [intuit/auto#973]
**Describe the bug**

While reviewing a pr in the learnstorybook repo, more specifically the design systems for developer tutorial in the distribute section, as the documentation progresses i'm told to use auto to generate the changelog with:
```bash
yarn auto changelog
```

Running said command yelds the following:
```bash
‼  warning   Could not find plugin: 
\learnstorybook-design-system-example\node_modules\@auto-it\core\dist\plugins\filter-non-pull-request.js
fatal: --is-ancestor takes exactly two commits
Error: Command failed: git merge-base --is-ancestor 7c25418fda716c54e47328c8667db434e9204fad 4f153ac8581d871fbeea3433b1c47e19c0dcadfe; echo $?
fatal: --is-ancestor takes exactly two commits

    at checkExecSyncError (child_process.js:621:11)
    at Object.execSync (child_process.js:657:15)
    at \learnstorybook-design-system-example\node_modules\@auto-it\core\dist\release.js:220:46
    at Array.filter (<anonymous>)
    at Release.getCommits (\learnstorybook-design-system-example\node_modules\@auto-it\core\dist\release.js:217:67)
    at async Release.getSemverBump (\learnstorybook-design-system-example\node_modules\@auto-it\core\dist\release.js:275:25)
    at async Auto.makeChangelog (\learnstorybook-design-system-example\node_modules\@auto-it\core\dist\auto.js:883:22)
    at async Auto.changelog (\learnstorybook-design-system-example\node_modules\@auto-it\core\dist\auto.js:538:9)
    at async run (\learnstorybook-design-system-example\node_modules\auto\dist\run.js:53:17)
    at async Object.main [as default] (\learnstorybook-design-system-example\node_modules\auto\dist\run.js:96:5) {
  status: 128,
  signal: null,
  output: [ null, '', 'fatal: --is-ancestor takes exactly two commits\n' ],
  pid: 6800,
  stdout: '',
  stderr: 'fatal: --is-ancestor takes exactly two commits\n'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Expected behavior**

Should yeld a changelog.

**Desktop (please complete the following information):**

- OS: windows 10

- Version 9.10.4

**Additional context**

I've looked up for a similar issue and did some digging in the documentation and i could not find any information about this. I have the code used in [this repo](https://github.com/jonniebigodes/design-systems-with-storybook-example) 

Feel free to provide feedback.

**Comment by hipstersmoothie** [comment]
can you give me an example repo?

**Comment by hipstersmoothie** [comment]
I cloned you repo and ran `yarn auto changelog` and it output a `changelog`. 

Can you run `yarn auto info` and give me the result?

**Comment by jonniebigodes** [comment]
@hipstersmoothie thanks for picking up on this.

Here's the output you requested:
```bash
‼  warning   Could not find plugin: C:\Users\MyUser\Myfolder\storybook_fork\storybook-related\learnstorybook-design-system-example\node_modules\@auto-it\core\dist\plugins\filter-non-pull-request.js

Environment Information:

"auto" version: v9.10.4
"node" version: v12.15.0


Project Information:

√ Repository:      jonniebigodes/design-systems-with-storybook-example (​https://github.com/jonniebigodes/design-systems-with-storybook-example​)
√ Author Name:     jonniebigodes
√ Author Email:    jonniebigodes@hotmail.com
√ Current Version: v0.1.0
× Latest Release:

√ Labels configured on GitHub project

GitHub Token Information:

√ Token:            [Token starting with e3e0]
√ Repo Permission:  admin
√ User:             jonniebigodes
√ API:              https://api.github.com (​https://api.github.com​)
√ Enabled Scopes:   repo
√ Rate Limit:       4997/5000

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by hipstersmoothie** [comment]
Can you run `git merge-base --is-ancestor 7c25418fda716c54e47328c8667db434e9204fad 4f153ac8581d871fbeea3433b1c47e19c0dcadfe`? on my machine this exits with `1` but doesn't cause the error above

<img width="1188" alt="Screen Shot 2020-02-22 at 4 36 26 PM" src="https://user-images.githubusercontent.com/1192452/75101443-7f256f00-5591-11ea-89e7-9bd77ebfe265.png">

**Comment by hipstersmoothie** [comment]
Could you also run `git --version`

**Comment by jonniebigodes** [comment]
Running `git --version ` gives me the following:
```bash
git version 2.17.0.windows.
```

And i did a bit of checking and checked the commit history. So `7c25418fda716c54e47328c8667db434e9204fad` is the latest commit and `4f153ac8581d871fbeea3433b1c47e19c0dcadfe` the initial.  So `25418fda716c54e47328c8667db434e9204fad` should be a descendant of it and based on the documentation it seems if the command runs exits with 1 it seems that git is not detecting it as a descendant.

**Comment by hipstersmoothie** [comment]
Hmmm odd. Windows must throw an error instead of exitting. I'll update the code to handle the case where an error is thrown.

**Comment by hipstersmoothie** [comment]
I think I have merged a fix that will address this for the most part. Feel free to re-open this issue if you are still experiencing it.

**Comment by adierkens** [comment]
:rocket: Issue was released in `v9.13.1` :rocket:

**Comment by jonniebigodes** [comment]
@hipstersmoothie and @adierkens  going to test it out tomorrow and let y'all know. Thanks for the time you put into this and i'll let you know.

**Comment by jonniebigodes** [comment]
@hipstersmoothie and @adierkens i bumped the version and with the update it works i get the message output and changelog addition:
```
‼  warning   Could not find plugin: \learnstorybook-design-system-example\node_modules\@auto-it\core\dist\plugins\filter-non-pull-request.js
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
i  info      New Release Notes
 #### ⚠️  Pushed to master

- changes up to distribute section  ([@jonniebigodes](https://github.com/jonniebigodes))
- a simple test added for link  ([@jonniebigodes](https://github.com/jonniebigodes))
- adding chromatic  ([@jonniebigodes](https://github.com/jonniebigodes))
- ignore storybook static  ([@jonniebigodes](https://github.com/jonniebigodes))
- changes for review chapter  ([@jonniebigodes](https://github.com/jonniebigodes))

#### Authors: 1

- [@jonniebigodes](https://github.com/jonniebigodes)
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
‼  warning   warning: LF will be replaced by CRLF in CHANGELOG.md.
The file will have its original line endings in your working directory.

fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
```
but i can live with this,  once again thank you for the speedy fix.

**Comment by hipstersmoothie** [comment]
Oh man that's still really ugly. cc @adierkens maybe can make this work on windows better

**Comment by jonniebigodes** [comment]
@hipstersmoothie like i said, it was working, but if you guys can improve it. Fine by me, i appreciate it.

**Comment by hipstersmoothie** [comment]
My new theory is that windows is interpreting this command differently than in `unix` envs:

```
git merge-base --is-ancestor 7c25418fda716c54e47328c8667db434e9204fad 4f153ac8581d871fbeea3433b1c47e19c0dcadfe; echo $?
```

it might be treating each of the following as args

- `7c25418fda716c54e47328c8667db434e9204fad`
- `4f153ac8581d871fbeea3433b1c47e19c0dcadfe;` (not a commit)
- `echo $?` (third commit?)

**Comment by hipstersmoothie** [comment]
I think I solved it. #1019 should have a fix. If you could install the canary version at some point and take a look that would be awesome (or @adierkens if he gets a chance)

**Comment by adierkens** [comment]
I was able to repro on my windows machine and #1019 seemed to clean up the logs. 

Before: 
```
PS C:\Users\adam\Developer\design-systems-with-storybook-example> yarn auto changelog
yarn run v1.16.0
$ C:\Users\adam\Developer\design-systems-with-storybook-example\node_modules\.bin\auto changelog
‼  warning   Could not find plugin: C:\Users\adam\Developer\design-systems-with-storybook-example\node_modules\@auto-it\core\dist\plugins\filter-non-pull-request.js
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
i  info      New Release Notes
 #### ⚠️  Pushed to master

- Change something  ([@adierkens](https://github.com/adierkens))

#### Authors: 1

- Adam Dierkens ([@adierkens](https://github.com/adierkens))
fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
‼  warning   warning: LF will be replaced by CRLF in CHANGELOG.md.
The file will have its original line endings in your working directory

fatal: --is-ancestor takes exactly two commits
fatal: --is-ancestor takes exactly two commits
Done in 9.40s.
PS C:\Users\adam\Developer\design-systems-with-storybook-example>
```


After:

```
PS C:\Users\adam\Developer\design-systems-with-storybook-example> yarn auto changelog
yarn run v1.16.0
$ C:\Users\adam\Developer\design-systems-with-storybook-example\node_modules\.bin\auto changelog
‼  warning   Could not find plugin: C:\Users\adam\Developer\design-systems-with-storybook-example\node_modules\@auto-canary\core\dist\plugins\filter-non-pull-request.js
‼  warning   Could not find plugin: npm
i  info      New Release Notes
 #### ⚠️  Pushed to `master`

- Change something else  ([@adierkens](https://github.com/adierkens))
- Change something  ([@adierkens](https://github.com/adierkens))

#### Authors: 1

- Adam Dierkens ([@adierkens](https://github.com/adierkens))
‼  warning   warning: LF will be replaced by CRLF in CHANGELOG.md.
The file will have its original line endings in your working directory

Done in 10.47s.
```

**Comment by adierkens** [comment]
:rocket: Issue was released in `v9.15.9` :rocket:
